### PR TITLE
Update quickstart guide to use KDD mode

### PR DESCRIPTION
**Pull request by caseydavenport**
Gets rid of the current etcd.yaml, which isn't production ready and trips people up in multi-master scenarios.

- [ ] Tests
- [ ] Documentation
- [ ] Release note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 7965b7d5bc07698ae2b0ca5c5d86f82bbda957b6

https://deploy-preview-2435--calico.netlify.com

File: master/getting-started/kubernetes/index.md [hunk 1]
```diff
@@ -60,49 +60,42 @@ To deploy a cluster suitable for production, refer to [Installation](installatio
...
-   ```
-   {: .no-select-button}
-
 1. Install {{site.prodname}} with the following command.
 
    ```
    kubectl apply -f \
-   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/calico.yaml
+   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml
+
```
Comment (lwr20) [inline master/getting-started/kubernetes/index.md]: unnecessary blank line?

File: master/getting-started/kubernetes/installation/hosted/etcd.yaml [hunk 2]
```diff
@@ -1,86 +0,0 @@
----
```
Comment (lwr20) [inline master/getting-started/kubernetes/installation/hosted/etcd.yaml]: It'd be nice to keep this file in the docs, even if we don't link to it - its used by CI.

Comment (caseydavenport) [reply master/getting-started/kubernetes/installation/hosted/etcd.yaml]: As discussed in Slack, let's drop this and adjust CI to use something else.

**Pull request merged**